tests/harness/cfg/cloud/aws/environments.yaml

```diff
@@ -34,19 +34,6 @@ environments:
     mandatory_requirements:
       - docker-local
 
-  - name: aws-kubeflow-orchestrator
```

Comment:
No more kubeflow orchestrator in zentest?

Reply:
This I've also wondered about, are there any plans to re-introduce these environment using the modular recipe?